### PR TITLE
RiotGames.RiotClient version 30.0.1.3715678

### PR DESCRIPTION
**Pull request by caioferndev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16141)

### CODE REVIEW

File: manifests/r/RiotGames/RiotClient/30.0.1.3715678/RiotGames.RiotClient.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate
+
+PackageIdentifier: RiotGames.RiotClient
```
Comment (OfficialEsco) [inline manifests/r/RiotGames/RiotClient/30.0.1.3715678/RiotGames.RiotClient.installer.yaml:3]:
```suggestion
PackageIdentifier: RiotGames.LeagueOfLegends
```

File: manifests/r/RiotGames/RiotClient/30.0.1.3715678/RiotGames.RiotClient.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: RiotGames.RiotClient
```
Comment (OfficialEsco) [inline manifests/r/RiotGames/RiotClient/30.0.1.3715678/RiotGames.RiotClient.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: RiotGames.LeagueOfLegends
```

File: manifests/r/RiotGames/RiotClient/30.0.1.3715678/RiotGames.RiotClient.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: RiotGames.RiotClient
+PackageVersion: 30.0.1.3715678
+PackageLocale: en-US
+Publisher: Riot Games, Inc.
+PackageName: LeagueOfLegends
```
Comment (OfficialEsco) [inline manifests/r/RiotGames/RiotClient/30.0.1.3715678/RiotGames.RiotClient.locale.en-US.yaml:7]:
```suggestion
PackageName: League of Legends
```

File: manifests/r/RiotGames/RiotClient/30.0.1.3715678/RiotGames.RiotClient.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: RiotGames.RiotClient
```
Comment (OfficialEsco) [inline manifests/r/RiotGames/RiotClient/30.0.1.3715678/RiotGames.RiotClient.yaml:3]:
```suggestion
PackageIdentifier: RiotGames.LeagueOfLegends
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Hmm can you rename the files and the `PackageIdentifier` to `RiotGames.LeagueOfLegends` ?

**Comment by OfficialEsco** [comment]
You need to rename the folder and files.
`Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\r\RiotGames\RiotClient\30.0.1.3715678', expected 'manifests\r\RiotGames\LeagueOfLegends\30.0.1.3715678'`

**Comment by hunnby** [comment]
My mistake, should be gathering these changes before commiting, haven't used git in a while.

**Comment by OfficialEsco** [comment]
> My mistake, should be gathering these changes before commiting, haven't used git in a while.

those changes are waaaaay too easy to miss so no worrys 😁 I hope you know you just hope you changed 1 out of 3 files tho 🙄

**Comment by denelon** [comment]
@hunnby no worries.

**Comment by hunnby** [comment]
This should be it... thanks for the patience 😄

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**